azext_iot/central/command_map.py

```diff
@@ -151,6 +151,12 @@ def load_central_commands(self, _):
             "get_device_twin",
         )
 
+    with self.command_group(
+        "iot central device c2d-message",
+        command_type=central_device_ops,
+    ) as cmd_group:
+        cmd_group.show_command("purge", "purge_c2d_messages")
```

Comment:
Should be  `cmd_group.command("purge", "purge_c2d_messages")` (not a show_command)